### ISSUE TITLE
Missing event to notify of changes in recommended view image size

### ISSUE THREAD

**Issue by Ralith** [KhronosGroup/OpenXR-Docs#19]
This is needed to allow applications to respond to changes in runtime configuration (e.g. SteamVR's application supersampling slider) without restarting or polling `xrEnumerateViewConfigurationViews` every frame. Applications should be encouraged to respond to such configuration changes dynamically so users can judge the visual impact of settings changes.

CC @rpavlik

**Comment by rpavlik** [comment]
Thanks for filing this! I've opened a corresponding issue (number 1150) within the working group.

My initial impression is that in the present spec, recommended view size is constant for the lifetime of the instance+system. I suspect this might be addressable via an extension.

**Comment by rpavlik** [comment]
This is being tracked in Khronos GitLab as issue 619.

**Comment by bjornbytes** [comment]
This can be closed now that `XR_EXT_view_configuration_views_change` and `XR_META_recommended_layer_resolution` both exist, right?

**Comment by rpavlik** [comment]
Yes, good catch, the XR_EXT_view_configuration_views_change extension is the one to use here.
